package.json

```diff
@@ -1,34 +1,39 @@
 {
-  "name": "xshare-api",
+  "name": "xshare-serverless",
   "version": "1.0.0",
   "description": "",
-  "main": "out/index.js",
+  "main": "./src/app.js",
   "scripts": {
-    "start": "node .",
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node ./src/app.tsx",
+    "dev": "concurrently \"tsc -w\" \"nodemon ./src/app.tsx\"",
     "prebuild": "npm install && tslint -c tslint.json -p tsconfig.json --fix",
     "build": "tsc",
```

Comment:
We should also remove the contents inside the outDir folder so that if we make file structure changes, there won't be non referenced files ghosting around in the outDir folder.

Reply:
ok I can do that